remote-working-etiquette.md

```diff
@@ -0,0 +1,30 @@
+# Remote working etiquette
+
+Working remotely is a great way to get focussed, balance work and life, or maybe just have a change of scenery and be inspired. However, it's really important that while we enjoy these benefits and seek our own state of super-productivity, we remain conscious of the effectiveness of our colleagues and the team around us 
+
+## Remote considerations
+
+There is no list of acceptable reasons for when it's "ok" to work remotely. You'll be the best judge of whether not being physically present/absent is going to positively or negatively impact the success of the team. Here are some questions you might want to factor in to that decision:
+
+Environment
+- Will I be able to have good virtual access to the team from my remote location (i.e. internet speed able to support a Google Hangout)?
+- Will my environment allow me to focus on my work (i.e. minimal distraction from media, people and of course pets)?
+- Will my environment allow me to communicate easily with colleagues (i.e. minimal background noise and the ability to speak clearly/openly)?
+- Will I be able to run all systems (i.e VM) necessary to complete my work and support the team?
+
+Commitment
+- Will you still be able to make your commitment to any meetings?
+- Will working remotely hinder your ability to hit your sprint commitments?
+
```

Comment:
Just a small observation for using first person narrative in the Environment section and second person narrative in the Commitment section. Probably sticking to one would be better?